GitExtUtils/FileSystemUtility.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.IO;
+
+namespace GitExtUtils
+{
+    public static class FileSystemUtility
+    {
+        /// <summary>
+        /// Returns the current working directory for display purposes or the exception message in the unlikely case of inability.
+        /// </summary>
+        public static string GetWorkingDirectoryNoEx()
```

Comment:
Please remove it. This has a single call site, and the return value of this method incoherent - "_get current directory or exception message_".

Reply:
It will also be reused when we create a wrapper for `FileSystem` which throws `ExternalOperationExceptions`.
I am always open for better names (not my forte).